docs/rust_wasi_markdown_parser/src/main.rs

```diff
@@ -0,0 +1,41 @@
+// Import our markdown parser library, crate
+extern crate pulldown_cmark;
+
+use pulldown_cmark::{html, Parser};
+
+// Import from the standard library, to allow reading CLI arguments, and from the file system
+use std::env;
+use std::fs;
+
+// Our entrypoint into our WASI module
+fn main() {
+
+    // Get the passed CLI arguments
+    let args: Vec<String> = env::args().collect();
```

Comment:
I think it'd be nice to use one of the existing arg parsing crates here, to get good help messages, etc. Using StructOpt, that could look roughly like this:

```rust

#[derive(StructOpt)]
pub struct Options {
    /// The markdown file to render
    #[structopt(parse(from_os_str))]
    filename: PathBuf,
}


fn main() {
    let options = Options::from_args();
   // and then use `options.filename`
  // [..]
}
```